src/drafts/MarkdownEditor/_FormattingTools.tsx

```diff
@@ -25,9 +25,14 @@ let hasRegisteredToolbarElement = false
  */
 export const FormattingTools = forwardRef<FormattingTools, {forInputId: string}>(({forInputId}, forwadedRef) => {
   useEffect(() => {
-    // requiring this module will register the custom element; we don't want to do that until the component mounts in the DOM
-    if (!hasRegisteredToolbarElement) require('@github/markdown-toolbar-element')
-    hasRegisteredToolbarElement = true
+    const fn = async () => {
+      if (hasRegisteredToolbarElement) return
+      // To prevent double-registering while the promise is pending, set it before importing
+      hasRegisteredToolbarElement = true
+      // requiring this module will register the custom element; we don't want to do that until the component mounts in the DOM
+      await import('@github/markdown-toolbar-element')
```

Comment:
I considered wrapping this `import()` call in a `try` and setting `hasRegisteredToolbarElement = false` on failure. But I don't think it's really beneficial to keep retrying if this fails -- this isn't an API request that might eventually succeed.